packages/ListBoxWithTags/stories/examples/FormElement.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import FormElement from "@paprika/form-element";
+import L10n from "@paprika/l10n";
+import ListBox, { useListBoxWithTags } from "../../src";
+
+const items = [
+  { label: "Heirloom scenester chillwave" },
+  { label: "Pickled paleo quinoa" },
+  { label: "Readymade hammock succulents" },
+];
+
+export default function FormElementExample() {
+  const { isSelected, filteredData, getSelectedOptions, ...moreUseListBoxWithTagsProps } = useListBoxWithTags({
+    key: "label",
+    defaultData: items,
+    defaultFilteredData: items,
+  });
+
+  return (
+    <L10n>
+      <FormElement isRequired>
+        <FormElement.Label>Lorem Hipsum Label</FormElement.Label>
+        <FormElement.Instructions>Scenester brunch instructions</FormElement.Instructions>
+        <FormElement.Content>
+          {a11yProps => (
+            <ListBox
+              {...moreUseListBoxWithTagsProps}
+              data-qa-anchor="listbox-tags-demo"
+              hasError
+              selectedOptions={getSelectedOptions()}
+            >
+              <ListBox.A11y {...a11yProps} />
```

Comment:
Since the `ListBoxWithTags` doesn't support an inline version, it would be possible to just pass these props to `<ListBoxWithTags.Trigger>`, but for consistency with `ListBox`, this same API is preferred.